src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/EFLComponentBasedApplication.cs

```diff
@@ -11,7 +11,7 @@ namespace Tizen.Applications.ComponentBased.Default
     /// <since_tizen> 6 </since_tizen>
     public class EFLComponentBasedApplication : ComponentBasedApplication
     {
-        private const string LogTag = "Tizen.Applications.ComponentBasedApplication";
+        private const string LogTag = "Tizen.Applications";
 
```

Comment:
Hello,
Why Application's code is changed? I think rebase or patch modify is required.
please check.
thank you.